### PR TITLE
Menu overlay score bug fix

### PR DESCRIPTION
**Pull request by Salvationdk**
## Description

Fixed overlay menu
## Screenshots (if appropriate):

menu
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: bot.user.js [hunk 1]
```diff
@@ -1148,10 +1148,12 @@ var userInterface = window.userInterface = (function () {
...
 
-            oContent.push('games played: ' + bot.scores.length);
-            oContent.push('avg score: ' + Math.round(
+				oContent.push('<br/>------------------');
+				oContent.push('games played: ' + bot.scores.length);
+				oContent.push('avg score: ' + Math.round(
                 bot.scores.reduce(function (a, b) { return a + b; }) / (bot.scores.length)));
-
+					oContent.push('------------------');
+					oContent.push('top 10 scores:');
```
Comment (ermiyaeskandary) [inline bot.user.js:1156]: ^^^

File: bot.user.js [hunk 2]
```diff
@@ -1148,10 +1148,12 @@ var userInterface = window.userInterface = (function () {
...
             if (bot.scores.length === 0) return;
 
-            oContent.push('games played: ' + bot.scores.length);
-            oContent.push('avg score: ' + Math.round(
+				oContent.push('<br/>------------------');
+				oContent.push('games played: ' + bot.scores.length);
+				oContent.push('avg score: ' + Math.round(
                 bot.scores.reduce(function (a, b) { return a + b; }) / (bot.scores.length)));
-
+					oContent.push('------------------');
```
Comment (ermiyaeskandary) [inline bot.user.js:1155]: ^^^

File: bot.user.js [hunk 3]
```diff
@@ -1148,10 +1148,12 @@ var userInterface = window.userInterface = (function () {
 
             if (bot.scores.length === 0) return;
 
-            oContent.push('games played: ' + bot.scores.length);
-            oContent.push('avg score: ' + Math.round(
+				oContent.push('<br/>------------------');
+				oContent.push('games played: ' + bot.scores.length);
+				oContent.push('avg score: ' + Math.round(
```
Comment (ermiyaeskandary) [inline bot.user.js:1153]: ^^^

File: bot.user.js [hunk 4]
```diff
@@ -1148,10 +1148,12 @@ var userInterface = window.userInterface = (function () {
 
             if (bot.scores.length === 0) return;
 
-            oContent.push('games played: ' + bot.scores.length);
-            oContent.push('avg score: ' + Math.round(
+				oContent.push('<br/>------------------');
+				oContent.push('games played: ' + bot.scores.length);
```
Comment (ermiyaeskandary) [inline bot.user.js:1152]: ^^^

File: bot.user.js [hunk 5]
```diff
@@ -1148,10 +1148,12 @@ var userInterface = window.userInterface = (function () {
 
             if (bot.scores.length === 0) return;
 
-            oContent.push('games played: ' + bot.scores.length);
-            oContent.push('avg score: ' + Math.round(
+				oContent.push('<br/>------------------');
```
Comment (ermiyaeskandary) [inline bot.user.js:1151]: ^^^

**Comment by ermiyaeskandary** [comment]
Approved...

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/256/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by ChadSki** [comment]
Hound's complaint comes from you using tabs instead of 4 spaces.

**Comment by ermiyaeskandary** [comment]
@Salvationdk 

> Hound's complaint comes from you using tabs instead of 4 spaces.

**Comment by ChadSki** [comment]
@ErmiyaEskandary I see you merged... Was the tabs/spaces thing fixed?

**Comment by ermiyaeskandary** [comment]
I locally pulled the merge, fixed it and then pushed :+1:

**Pull request merged**